hardhat.config.ts

```diff
@@ -76,4 +77,9 @@ export default {
       hardhat: 1,
     },
   },
+  gasReporter: {
+    enabled: process.env.REPORT_GAS ? true : false,
```

Comment:
Although this is how they suggest using it in their readme I can do this if you'd like.

Reply:
I think it's nicer to destructure it at the beginning for consistency within this file. Otherwise the line above is misleading, since you may think that only the variables destructured there are ever used in the configs.

Reply:
Done!